src/util/index.ts

```diff
@@ -16,6 +16,8 @@
 	along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
+// NOTE: !! DO NOT REORDER THE IMPORTS !!
```

Comment:
why not?

Reply:
> why not?

because if you do, it causes errors that will cost you many hours trying to debug, been there done that because of prettier auto formatting the imports lmao

Reply:
I see...